test/s3_test.erl

```diff
@@ -39,10 +38,6 @@ get_put() ->
     ?assertMatch({ok, _}, s3:put(bucket(), <<"foo-copy">>, <<>>, "text/plain",
                                  5000, [{"x-amz-copy-source", bucket() ++ "/foo"}])).
 
-permission_denied() ->
-    ?assertEqual({error, {"AccessDenied", "Access Denied"}},
```

Comment:
Has something broke the test?

Reply:
There is no access checks on fakes3, this test depended on real S3.